ui/src/containers/NodeVolumes.js

```diff
@@ -33,8 +39,12 @@ import {
 const VolumeTable = styled.div`
   flex-grow: 1;
   margin-top: ${padding.small};
-  .sc-table-column-cell-container-action {
-    justify-content: center;
+  /*solve the tooltip display issue*/
+  .ReactVirtualized__Grid {
+    overflow: visible !important;
```

Comment:
The use of '!important' should be avoided but I can not find a better solution. We can leave it here